### ISSUE TITLE
Test on Embedded platforms

### ISSUE THREAD

**Issue by wiresio** [wiresio/wotjs#3]
... not just Ubuntu VM

**Comment by rzr** [comment]
Like TizenRT ?

**Comment by wiresio** [comment]
Yes, for example. Or even bare metal on STM4 or ESP32. Would be interesting to see if we could run a real embedded WoT stack.

**Comment by rzr** [comment]
Interesting, I have a STM32F3 and ESP32 to give a try...
but 1st please licence the code by adding a licence file.

For the context I am the one behind:
https://social.samsunginter.net/@rzr/100969945665369600

**Comment by wiresio** [comment]
Some links about compiling for ESP32:
https://github.com/pando-project/iotjs/issues/536
https://dl.espressif.com/doc/esp-idf/latest/get-started/linux-setup.html
